static/js/components/EducationTab.js

```diff
@@ -103,26 +111,33 @@ class EducationTab extends ProfileTab {
   }
 
   printExistingEducations(level) {
-    const {profile} = this.props;
-    return profile['education'].map(education => {
+    const { profile, errors } = this.props;
+    let educationRow = (education, index) => {
       if (education.degree_name === level.value && 'id' in education) {
-        return <Grid key={"education-row-" + education.id} className="existing-education-grid">
+        let deleteEntry = () => this.deleteEducationEntry(index);
+        let editEntry = () => (
+          this.openEditEducationForm(education.degree_name, education.id)
+        );
+        let validationAlert = () => {
+          if (_.get(errors, ['education', index])) {
+            return <IconButton name="error" onClick={editEntry} />;
+          }
+        };
+        return <Grid key={`education-row-${education.id}`} className="existing-education-grid">
           <Cell col={3}>{this.educationLevelLabels[education.degree_name]}</Cell>
           <Cell col={7}>{education.graduation_date}</Cell>
-          <Cell col={1}>
-            <Button onClick={this.openEditEducationForm.bind(this, education.degree_name, education.id)}>
-              <Icon name="edit"/>
-            </Button>
-          </Cell>
-          <Cell col={1}>
-            <Button ><Icon name="delete"/></Button>
+          <Cell col={2} className="education-icons">
+            {validationAlert()}
+            <IconButton name="edit" onClick={editEntry} />
+            <IconButton name="delete" onClick={deleteEntry} />
           </Cell>
         </Grid>;
       }
-    });
+    };
+    return profile['education'].map(educationRow);
   }
 
-  openEditEducationForm(level, educationId){
+  openEditEducationForm = (level, educationId) => {
```

Comment:
yeah, exactly. when we declare an arrow function in the body of a class declaration `this` refers to the object instance at runtime.